### PR TITLE
Add an option to set window title to the command line

### LINKED ISSUE #3129: Is there a way to modify the main window title?
We are working in a multi-user environment with multiple instances of spyder opened. It would be convenient to show the session name (or any string) in the window title. Is there already a method to to this? If not, I can add a command line option to enable this.

### PR DESCRIPTION
**Pull request by eendebakpt**
Fixes #3129

### CODE REVIEW

File: spyderlib/app/cli_options.py [hunk 1]
```diff
@@ -37,5 +37,7 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--windowtitle', type=str, default=None,
```
Comment (ccordoba12) [inline spyderlib/app/cli_options.py]: Could add an hyphen between `window` and `title`? So this option should be

    --window-title

File: spyderlib/app/cli_options.py [hunk 2]
```diff
@@ -37,5 +37,7 @@ def get_options():
...
+                      help="String to show in main window")
```
Comment (ccordoba12) [inline spyderlib/app/cli_options.py]: This help text should read

    String to show in the main window title

**Comment by ccordoba12** [comment]
Other than my minor comments, I think this is ready :-)

**Comment by eendebakpt** [comment]
Ok, code is updated with the suggestions.

File: spyderlib/app/spyder.py [hunk 3]
```diff
@@ -439,6 +439,9 @@ def signal_handler(signum, frame=None):
                                                sys.version_info[1])
         if DEBUG:
             title += " [DEBUG MODE %d]" % DEBUG
+        if not options.window_title is None:
```
Comment (Nodd) [inline spyderlib/app/spyder.py]: `if options.window_title is not None:` is more commonly used in python, because it looks like an english sentence.

Comment (eendebakpt) [inline spyderlib/app/spyder.py]: Ok, changed to code for this

**Comment by ccordoba12** [comment]
@eendebakpt, thanks a lot for your contribution! :-)

**Pull request merged**